utils/FileHandler.hpp

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include "shapes/Shapes.hpp"
+#include "Canvas.hpp"
+#include "exceptions.hpp"
+#include "utils/algorithms.hpp"
+#include <QString>
+#include <QChar>
+#include <QMap>
+#include <QColor>
+#include <QFont>
+#include <QFile>
+#include <QFileDialog>
+#include <QTextStream>
+
+namespace jbrush {
+
+namespace jconstants {
+// Base extension for files used to load from/save to
+const QString FILE_EXTENSION = ".txt";
```

Comment:
:(

Reply:
Oh haha I only had it that way so I could keep testing.  I'll switch it back before my next commit after testing

Reply:
:)

Reply:
:)